environments.yml

```diff
@@ -56,6 +56,10 @@ scrapers:
     paramstore: /bobsled/backups/AWS_ACCESS_KEY_ID
   - variable: AWS_SECRET_ACCESS_KEY
     paramstore: /bobsled/backups/AWS_SECRET_ACCESS_KEY
+  - variable: AWS_DEFAULT_REGION
+    paramstore: /bobsled/backups/AWS_DEFAULT_REGION
```

Comment:
Any reason we can't just set this here? Does it have to be a parameter store value?

Reply:
I thought of possible case in the future where  region might change and we have it hardcoded here

Reply:
This is an interesting trade-off! The amount of effort to create and merge a PR here is relatively low, and it is _arguably_ slightly faster to load from a file instead of ParameterStore. But the performance benefit is infinitesimally small, and if the value is already in ParameterStore, this is relatively harmless. I don't think there's a wrong answer here either way, but just wanted to call out a spot where we _didn't_ need ParameterStore.

Reply:
Understood, thanks for clarifying.